CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateRacing.cpp

```diff
@@ -209,15 +215,19 @@ HeatStroke::GameObject* Kartaclysm::StateRacing::GenerateRacer
 	return pRacer;
 }
 
+// TODO - move selection of AI kart/drivers to StatePlayerSelectionMenu
+//			Waiting for StatePlayerSelectionMenu to be changed to allow for determining number of AI racers
```

Comment:
Could you not move it there for this task and hardcode a fake number of AI competitors? This seems like a lot of changes to StateRacing that will ultimately get taken out and put into another class later. It was understandable when AI was hacked together that things were messy, but no sense adding onto the mess now if it can start to be cleaned up. I'm also very wary of our track record with "TODO" statements that they may never get done